pkg/controller/nginxingress/nginx_ingress_controller_test.go

```diff
@@ -892,6 +892,7 @@ func TestToNginxIngressConfig(t *testing.T) {
 	FakeDefaultSSLCertNoNamespace := getFakeDefaultSSLCert("fake", "")
 
 	FakeDefaultBackend := approutingv1alpha1.NICNamespacedName{"fakename", "fakenamespace"}
+	FakeCustomErrors := []int32{404, 503}
```

Comment:
add more test cases. Prove it works for outliers. What if they pass in an empty array? What if they pass in 1?